src/types.rs

```diff
@@ -141,10 +141,9 @@ pub struct ServiceObject {
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct StateObject {
-    pub attributes: std::collections::HashMap<String, String>,
+    pub attributes: Value,
```

Comment:
HA api starting returning bools in this response so it was probably naive to make it Map<String,String> to begin with.

Reply:
Is this just a single level of `Map`? Perhaps it'd be safer to use `Map<String, Value>`?

Reply:
Yeah thats probably reasonable.